nesta/core/routines/arxiv/arxiv_lolvelty.py

```diff
@@ -0,0 +1,54 @@
+from nesta.core.luigihacks.estask import ElasticsearchTask
+from nesta.core.luigihacks.misctools import find_filepath_from_pathstub as f3p
+import luigi
+import logging
+from datetime import datetime as dt
+from nesta.core.orms.orm_utils import setup_es, get_es_ids
+
+
+class ArxivElasticsearchTask(ElasticsearchTask):
+    def done_ids(self):
+        es_mode = 'dev' if self.test else 'prod'
+        es, es_config = setup_es(es_mode, self.test,
+                                 drop_and_recreate=False,
+                                 dataset=self.dataset,
+                                 increment_version=False)
+        ids = get_es_ids(es, es_config, size=10000,
+                         query={"query": {"bool":
```

Comment:
reminder to change this!

Reply:
Thanks, i've changed to an `exists` query which returns 99.99% of the ids, which should be fine!